instaclustr/resource_cluster.go

```diff
@@ -352,7 +352,7 @@ func resourceCluster() *schema.Resource {
 								Type: schema.TypeString,
 							},
 							Optional: true,
-							ForceNew: true,
+							ForceNew: false,
```

Comment:
We might want to update the registry documentation to mention that this field is updatable, like we do for node_size, kafka_rest_proxy_user_password and kafka_schema_registry_user_password

Reply:
I think I created the branch on the instaclustr repo instead of forked repo. I will close this PR and create a new one from my fork

Reply:
Updated the `docs/resources/cluster.md` documentation file